README.md

```diff
@@ -4,79 +4,266 @@ Boost is a tool for Filecoin storage providers to manage data storage and retrie
+
+```
+lotus-miner actor control set --really-do-it $PUBMSG_WALLET
 ```
-LOTUS_PATH=~/.lotus
-LOTUS_MINER_PATH=~/.lotusminer
 
-rm -rf ~/.lotus ~/.lotusminer
+11. Add funds into the Market Actor escrow for the client and Collateral wallets
+
 ```
+lotus wallet market add --from $DEFAULT_WALLET --address $CLIENT_WALLET 5
+lotus wallet market add --address $COLLAT_WALLET 5
+```
+
+12. Initialize Boost / Create Boost repository
 
+```
+boostd -vv init \
+  --api-sealer=$MINER_API_INFO \
+  --api-sector-index=$MINER_API_INFO \
+  --wallet-publish-storage-deals=$PUBMSG_WALLET \
+  --wallet-deal-collateral=$COLLAT_WALLET \
+  --max-staging-deals-bytes=2000000000
+```
 
-3. Create Boost repository
+13. Build the Web UI
+```
+make react
+```
+
+14. Edit config to set a fixed listen address
+
+Edit `~/.boost/config.toml`
+
+Set the port in the `ListenAddresses` key to `50000`
+```
+[Libp2p]
+  ListenAddresses = ["/ip4/0.0.0.0/tcp/50000", "/ip6/::/tcp/0"]
+```
+
+15. Run Boost
+```
+boostd -vv run
+```
+
+Note the peer ID of the boost instance:
+```
+2022-06-10T09:32:28.819Z        INFO    boostd  boostd/run.go:114       Boost libp2p node listening     {"maddr": "{12D3KooWQNNWNiJ1mieEk9EHjDVF2qBc1FSjJGEzwjnMJzteApaW: [/ip4/172.17.0.2/tcp/50000 /ip4/127.0.0.1/tcp/50000]}"}
+```
+In this example: `12D3KooWQNNWNiJ1mieEk9EHjDVF2qBc1FSjJGEzwjnMJzteApaW`
 
+14. Set the peer ID and multi-address of the miner on chain
 ```
-export $(lotus auth api-info --perm=admin)
-export $(lotus-miner auth api-info --perm=admin)
+lotus-miner actor set-peer-id <peer id>
+lotus-miner actor set-addrs /ip4/127.0.0.1/tcp/50000
+```
+
+16. Open the Web UI
+
+Open http://localhost:8080 to see the Boost UI
 
-boostd --vv init \
-       --api-sealer=`lotus-miner auth api-info --perm=admin` \
-       --api-sector-index=`lotus-miner auth api-info --perm=admin` \
-       --wallet-publish-storage-deals=`lotus wallet new bls` \
-       --wallet-deal-collateral=`lotus wallet new bls` \
-       --max-staging-deals-bytes=50000000000
+### Make a deal with Boost
+
+1. Initialize the Boost client
+```
+boost init
 ```
 
-4. Run the Boost daemon service
+This will output the address of the wallet (it's safe to run the init command repeatedly).
 
+2. Send funds to the client wallet
+```
+lotus send --from=$DEFAULT_WALLET <client wallet> 10
 ```
-export $(lotus auth api-info --perm=admin)
 
-boostd --vv run
+3. Follow the guide at https://boost.filecoin.io/tutorials/how-to-store-files-with-boost-on-filecoin
+
+Note that above you already ran a command to export FULLNODE_API (and point it to your local devnet lotus daemon).
+
+Note also that the provider address is `t01000` and you will need to supply an appropriate `--storage-price` when using `boost deal` since the devnet has a minimum price. Alternatively, using "Settings" in the Boost web UI to set the deal price to zero.
+
+## Running Boost devnet in Docker
+
+### Building Docker images
```

Comment:
In the future, I would suggest pushing images to dockerhub and skipping this step for new users to start quickly without the building step.

Reply:
> In the future, I would suggest pushing images to dockerhub and skipping this step for new users to start quickly without the building step.

Agree, but I am not sure we should document the pushing here really... Pushing of images is going to happen within the team, by whoever has access to the Docker Registry.

---

We should make sure we have support to build Boost from source, because most developers on the team would be running a dev version of the Boost image, and sometimes even of Lotus images.